src/Router.php

```diff
@@ -5,36 +5,93 @@
 class Router
 {
     protected $_routeList;
-    
+
     protected static $instance;
-    
+
     public static function getInstance()
     {
         if (!isset(static::$instance)) {
             static::$instance = new static;
         }
-        
+
         return static::$instance;
     }
-    
+
+    /**
+     * Loads the location of the routes file.
+     *
+     * @return string
+     */
+    protected function getRouteFileLocation()
+    {
+        $dir = getenv('HOME') . '/.volcano-sdk';
```

Comment:
@cthos I'm thinking it would be useful to be able to customize the path at which the route file is stored.

Would it make sense to make that accessible via something like `VolcanoSDK\Router::setRouteFileLocation($path);`